lib/docker2aci.go

```diff
@@ -167,24 +166,28 @@ func SquashLayers(images []acirenderer.Image, aciRegistry acirenderer.ACIRegistr
 	}
 
 	squashedFilename := getSquashedFilename(parsedDockerURL)
-	squashedImagePath := path.Join(outputDir, squashedFilename)
+	squashedImagePath := filepath.Join(outputDir, squashedFilename)
 
-	squashedImageFile, err := os.Create(squashedImagePath)
+	squashedTempFile, err := ioutil.TempFile(outputDir, "squashedFilename-")
```

Comment:
Do you think that the tempFile should be removed on error?
Perhaps it can be created inside the tmpDir so it can be removed from who created it.